forms-flow-web/src/components/Draft/ViewDraft.js

```diff
@@ -93,15 +92,6 @@ const ViewDraft = React.memo(() => {
         >
           <View page="draft-detail" />
         </Tab>
-        <Tab
```

Comment:
Keep the process Diagram. That is fine , Inicates which flow will trigger on Submission of drat